driver.sh

```diff
@@ -87,6 +87,12 @@ setup_variables() {
       qemu_cmdline=( -cpu cortex-a57
                      -drive "file=images/arm64/rootfs.ext4,format=raw"
                      -append "console=ttyAMA0 root=/dev/vda" )
```

Comment:
I strongly recommend adding "earlycon" here to catch the early boot console bits from arm64 boots. (And all the -append lines should have "loglevel=8" added to make sure the boot is as verbose as possible.)